core/blockchain/tx_utils.py

```diff
@@ -16,9 +16,12 @@ class Transaction(object):
             'value': ...
          }`
     """
-    def __init__(self, key: str, value: object) -> None:
+    def __init__(self, key: str, value: str) -> None:
         self.key = key
         self.value = value
 
-    def __repr__(self) -> dict:
+    def get_tx(self) -> dict:
+        # print(self.key)
+        # print(self.value)
+        # print('jjj', {TxEnum.KEY.name: [self.key], TxEnum.CONTENT.name: self.value})
```

Comment:
I presume you were using these print statements to test? Please delete when you're done testing.